cookbook/agents/input_and_output/input_schema_on_agent.py

```diff
@@ -24,15 +24,28 @@ class ResearchTopic(BaseModel):
     input_schema=ResearchTopic,
 )
 
-# Pass a dict that matches the input schema
-hackernews_agent.print_response(
-    input={
-        "topic": "AI",
-        "focus_areas": ["AI", "Machine Learning"],
-        "target_audience": "Developers",
-        "sources_required": "5",
-    }
+
+agent_os = AgentOS(
+    id="agentos-demo",
+    agents=[hackernews_agent],
 )
+app = agent_os.get_app()
+
+
+if __name__ == "__main__":
+    agent_os.serve(app="input_schema_on_agent:app", port=7777)
+
+
+
+# # Pass a dict that matches the input schema
+# hackernews_agent.print_response(
+#     input={
+#         "topic": "AI",
+#         "focus_areas": ["AI", "Machine Learning"],
+#         "target_audience": "Developers",
+#         "sources_required": "5",
+#     }
+# )
 
 # Pass a pydantic model that matches the input schema
 # hackernews_agent.print_response(
```

Comment:
Leftovers? this cookbook is not related to the OS, we probably shouldn't add it here